CHANGELOG.md

```diff
@@ -1,8 +1,7 @@
 # CHANGELOG
 All notable changes to this project will be documented in this file based on the [Keep a Changelog](http://keepachangelog.com/) Standard. This project adheres to [Semantic Versioning](http://semver.org/).
 
-
-## Unreleased
+## [1.0.0](https://github.com/elastic/ecs/compare/v1.0.0-beta2...v1.0.0)
```

Comment:
No unreleased section?

I would also expect the code generator to show up in this one?

Reply:
Ah right, I have an upcoming PR to create CHANGELOG.next.md ;-)

Reply:
Got it